src/test/resources/data/bsearch1.json

```diff
@@ -0,0 +1,675 @@
+                        "lte":"2021-04-06T17:42:53.295Z",
+                        "format":"strict_date_optional_time"
+                      }
+                    }
+                  }
+                ],
+                "should":[
+
+                ],
+                "must_not":[
+
+                ]
+              }
+            }
+          },
+          "preference":1617730961756
+        }
+      },
+      "options":{
+        "strategy":"ese"
+      }
+    },
+    {
+      "request":{
+        "params":{
+          "index":"kibana_sample_data_ecommerce",
+          "body":{
+            "aggs":{
+              "2":{
+                "date_histogram":{
+                  "field":"order_date",
+                  "fixed_interval":"3h",
+                  "time_zone":"Europe/Berlin",
+                  "min_doc_count":1
+                },
+                "aggs":{
+                  "3":{
+                    "terms":{
+                      "field":"category.keyword",
+                      "order":{
+                        "1":"desc"
+                      },
+                      "size":5
+                    },
+                    "aggs":{
+                      "1":{
+                        "sum":{
+                          "field":"total_quantity"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "size":0,
+            "fields":[
+              {
+                "field":"customer_birth_date",
+                "format":"date_time"
+              },
+              {
+                "field":"order_date",
+                "format":"date_time"
+              },
+              {
+                "field":"products.created_on",
+                "format":"date_time"
+              }
+            ],
+            "script_fields":{
+
+            },
+            "stored_fields":[
+              "*"
+            ],
+            "runtime_mappings":{
+
+            },
+            "_source":{
+              "excludes":[
+
+              ]
+            },
+            "query":{
+              "bool":{
+                "must":[
+
+                ],
+                "filter":[
+                  {
+                    "match_all":{
+
+                    }
+                  },
+                  {
+                    "range":{
+                      "order_date":{
+                        "gte":"2021-03-30T17:42:53.295Z",
+                        "lte":"2021-04-06T17:42:53.295Z",
+                        "format":"strict_date_optional_time"
+                      }
+                    }
+                  }
+                ],
+                "should":[
+
+                ],
+                "must_not":[
+
+                ]
+              }
+            }
+          },
+          "preference":1617730961756
+        }
+      },
+      "options":{
+        "sessionId":"3e0ee321-2b66-4da9-a956-23a9fbf07289",
```

Comment:
Should sessionId be taken from any of previous requests? Does hardcoded value makes the scenario less effective?